src/components/validations/validation-icon.component.js

```diff
@@ -48,22 +50,31 @@ const ValidationIcon = ({
       role="tooltip"
       aria-label={validationMessage}
       onClick={onClick}
+      onMouseOver={() => setHasHover(true)}
+      onMouseLeave={() => setHasHover(false)}
+      onFocus={() => setHasHover(true)}
+      onBlur={() => setHasHover(false)}
     >
-      <Icon
-        key={`${validationType}-icon`}
-        tooltipType={validationType}
-        tooltipMessage={validationMessage}
-        tooltipVisible={
-          hasFocus || hasMouseOver || groupHasFocus || groupHasMouseOver
-        }
+      <Tooltip
         type={validationType}
-        size={size}
-        tabIndex={tabIndex}
-        tooltipPosition={tooltipPosition}
-        tooltipAlign="center"
-        isThemeModern
+        message={validationMessage}
+        isVisible={
+          hasFocus ||
+          hasMouseOver ||
```

Comment:
`hasMouseOver` and `hasHover` are the same thing I think

Reply:
Oh sorry, `hasMouseOver` is set to `groupHasMouseOver` so I think you can remove one of them